src/sentry/api/endpoints/user_notification_fine_tuning.py

```diff
@@ -135,6 +143,16 @@ def _handle_put_reports(user, data):
             elif not enabled:
                 value.add(org_id)
 
+            if double_write_enabled:
```

Comment:
would be good to add a comment here explaining what's going on here / when this should be removed.